internal/orchestrationapi/orchestrationapi.go

```diff
@@ -334,15 +334,15 @@ func (orcheEngine orcheImpl) gatherDevicesScore(candidates []dbhelper.ExecutionC
 
 			if err != nil {
 				log.Println("[orchestrationapi] cannot getting score from :", cand.Endpoint[0], "cause by", err.Error())
-				scores <- deviceInfo{endpoint: cand.Endpoint[0], score: float64(0.0), id: cand.Id}
+				scores <- deviceInfo{endpoint: cand.Endpoint[0], score: float64(0.0), id: cand.ID}
 				return
 			}
 			log.Printf("[orchestrationapi] deviceScore")
-			log.Printf("candidate Id       : %v", cand.Id)
+			log.Printf("candidate Id       : %v", cand.ID)
```

Comment:
The variable ID comes from ExecutionCandidate structure from helper.go file. This file generated the following lint warning
internal/db/helper/helper.go:37:2: struct field Id should be ID
To remove this warning, the variable was changed to ID from Id. Hence the change in variable. Log message it can be reverted if required :)

Reply:
Done!